konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/bbox/point/BoundingBoxToPointStep.java

```diff
@@ -35,6 +35,7 @@
  * the output will be a single value; if the input is a list, the output will be a list.<br>
  */
 @Data
+@Builder
```

Comment:
No builder: we're standardizing on fluent API everywhere - https://github.com/KonduitAI/konduit-serving/issues/390